test/Conversion/ATenToXTen/aten_to_xten_resnet_torchvision.mlir

```diff
@@ -0,0 +1,279 @@
+//===- aten_resnet50_torchvision.mlir ---------------------------------*- MLIR -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2022 Xilinx Inc.
+//
+//===----------------------------------------------------------------------===//
+
+// ResNet50 from Torchvision, converted to MLIR via torch-mlir (op decomposition disabled)
```

Comment:
Given only softmax is checked, it is not clear to me what the test is intended to add above `aten_to_xten_softmax.mlir`. Is your plan that additional checks would be added in future PRs so one can verify that everything is converted in this one overview rather than the sum of smaller tests?

Reply:
Yes, I added it as a sort of integration test, to add more tests to it later and to make sure it doesn't break when we run it on the whole ResNet input. I can see how it'll be difficult to remember updating this test to keep it useful, I can get rid of it if you like.